app/views/portal/show.rb

```diff
@@ -751,15 +722,14 @@ def simple_rights_label_data(rights)
       end
     end
 
-    def creator_title
-      document.fetch('agents.prefLabel', []).first || render_document_show_field_value(document, 'dcCreator')
-    end
-
     # iiif manifests can be derived from some dc:identifiers - on a collection basis or an individual item basis - or from urls
-    def iiif_manifesto(identifier, collection)
+    def iiif_manifesto(document)
```

Comment:
Assignment Branch Condition size for iiif_manifesto is too high. [21.21/15]<br>Method has too many lines. [21/10]